traffic_ops/app/lib/API/Iso.pm

```diff
@@ -203,18 +215,21 @@ sub generate {
 
 	my $iso_url = join( "/", $config->{'to'}{'base_url'}, $iso_dir, $iso_file_name );
 
-	my $response;
-	$response->{isoURL} = $iso_url;
+	my $response = {
+		isoName => $iso_file_name,
+		isoURL => $iso_url,
+	};
 
-	return $self->success( $response, "Generate ISO was successful." );
```

Comment:
i think this change is a problem. you can't just return the response hash you have to wrap it in a 200 which the success method does

Reply:
or wait, nm, i see on line 83 you are doing it

return $self->success( $response, "Generate ISO was successful." );

Reply:
This is code I moved to a new method.  You can see the success getting returned here:  https://github.com/dneuman64/incubator-trafficcontrol/blob/cb149066027e4a7f730353f86356c77be86590e4/traffic_ops/app/lib/API/Iso.pm#L82